src/uct/ib/rc/accel/rc_mlx5.inl

```diff
@@ -504,6 +504,36 @@ uct_rc_mlx5_common_post_send(uct_rc_mlx5_iface_common_t *iface, int qp_type,
     uct_rc_txqp_posted(txqp, &iface->super, res_count, fm_ce_se & MLX5_WQE_CTRL_CQ_UPDATE);
 }
 
+static UCS_F_ALWAYS_INLINE void uct_rc_mlx5_txqp_inline_iov_post(
+        uct_rc_mlx5_iface_common_t *iface, uct_rc_txqp_t *txqp,
+        uct_ib_mlx5_txwq_t *txwq, const uct_iov_t *iov, size_t iovcnt,
+        uint8_t am_id)
+{
+    struct mlx5_wqe_ctrl_seg *ctrl;
+    struct mlx5_wqe_inl_data_seg *inl;
+    uct_rc_mlx5_hdr_t *rch;
+    size_t wqe_size, ctrl_size, iov_length;
+    void *next_seg;
+    unsigned fm_ce_se;
+
+    ctrl            = txwq->curr;
+    ctrl_size       = sizeof(*ctrl);
+    next_seg        = UCS_PTR_BYTE_OFFSET(ctrl, ctrl_size);
+    next_seg        = uct_ib_mlx5_txwq_wrap_exact(txwq, next_seg);
```

Comment:
you could use `uct_ib_mlx5_txwq_wrap_none` here, because ctrl segment size is 16 bytes. 
`uct_ib_mlx5_txwq_wrap_exact` will be needed when you add AV here for dc which takes 48 bytes.